packages/xcm-util/common/index.ts

```diff
@@ -151,12 +200,25 @@ export function location(
   };
 }
 
+/**
+ * Creates a fungible asset representation.
+ *
+ * @param amount - The amount of the fungible asset.
+ * @returns An object representing the fungible asset.
+ */
 export function fungible(amount: bigint) {
   return {
     fungible: amount,
   };
 }
 
+/**
+ * Creates a non-fungible asset representation.
+ *
+ * @param id - The identifier for the non-fungible asset, which can be a bigint or a string.
```

Comment:
Explain that a bigint will be an `{index: bigint-value}` and a string will be plugged into the corresponding `array*` variant.